nixos/modules/services/monitoring/monit.nix

```diff
@@ -36,11 +36,16 @@ in
     ];
 
     systemd.services.monit = {
-      description = "Monit system watcher";
-      after = [ "network-interfaces.target" ];
+      description = "Pro-active monitoring utility for unix systems";
```

Comment:
While this is a good description, what monit does it obscures the name if on the boot screen "Pro-active monitoring utility for unix systems started"  appears.